electrum/bitcoin.py

```diff
@@ -428,7 +428,7 @@ def address_to_hash(addr: str, *, net=None) -> str:
         raise BitcoinException(f"invalid bitcoin address: {addr}")
     witver, witprog = segwit_addr.decode(net.SEGWIT_HRP, addr)
     if witprog is not None:
-        return witver, bytes(witprog)
+        return net.SEGWIT_HRP, bytes(witprog)
```

Comment:
now this function is returning both ints and strings, which is not so nice. at least fix the function type signature ;)